### PR TITLE
Implemented task 1

### PR DESCRIPTION
**Pull request by Divyajg**

### CODE REVIEW

File: javascript/javascript1/week1/Homework1/homework.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/* Age - ify*/
+let yearOfBirth;
+let yearFuture;
```
Comment (TITHI1244) [inline javascript/javascript1/week1/Homework1/homework.js:3]: Why did you use `let`? `const` would have been a better choice, right? These 2 variables are not going to change, so. And also assign the variables while declaration. You do not need 2 lines for that :)

File: javascript/javascript1/week1/Homework1/homework.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+yearOfBirth = 1991;
+yearFuture = 2067;
+
+let age = (yearFuture - yearOfBirth);
```
Comment (TITHI1244) [inline javascript/javascript1/week1/Homework1/homework.js:7]: Again `let` :)

File: javascript/javascript1/week1/Homework1/homework.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+console.log(`Your will be ${age} years old in ${yearFuture}`);
```
Comment (TITHI1244) [inline javascript/javascript1/week1/Homework1/homework.js:8]: I think, you meant 'your age' inside console.log, right?

File: javascript/javascript1/week1/Homework1/homework.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+/*Goodboy - Oldboy */
+
+let dogYearOfBirth;
+let dogYearFuture;
+let dogYear;
+let shouldShowResultInDogYears;
+
+dogYearOfBirth = 2016;
+dogYearFuture = 2049;
+
```
Comment (TITHI1244) [inline javascript/javascript1/week1/Homework1/homework.js:19]: Same as before :)

File: javascript/javascript1/week1/Homework1/homework.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+dogYear = (dogYearFuture - dogYearOfBirth);
+
+/* shouldShowResultInDogYears should be true or false */
+shouldShowResultInDogYears = false;
+
+if (shouldShowResultInDogYears == true) {
+    console.log(`Your dog will be ${(1/7)*dogYear} years old in ${dogYearFuture} `);
```
Comment (TITHI1244) [inline javascript/javascript1/week1/Homework1/homework.js:26]: You could also do this calculation of dogyear while assigning dogYear variable. It seems cleaner to me :)

File: javascript/javascript1/week1/Homework1/homework.js [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+for (i = 0; i <= (buyer.length - 1); i++) {
+    volumeInCubicMeters[i] = houseWidth[i] * houseHeight[i] * houseDepth[i];
+    housePrice[i] = volumeInCubicMeters[i] * 2.5 * 1000 + gardenSizeInM2[i] * 300;
+    console.log(`Price of house bought by ${buyer[i]} is ${housePrice[i]}`);
+
+    if (houseCost[i] > housePrice[i]) {
+        console.log(` ${buyer[i]} is paying more than the Usual price.`);
+    } else {
+        console.log(`${buyer[i]} is paying less than the Usual price`);
+
```
Comment (TITHI1244) [inline javascript/javascript1/week1/Homework1/homework.js:53]: Very smart way of using array here :)

**Review by TITHI1244 — APPROVED** [review]
Very well done :) Remember to use `const` instead of `let` as much as you can. If the variable is changing some time later, then only use `let`. Keep coding

File: javascript/javascript1/week1/Homework1/homework.js (hunk 1 above)
Comment (Divyajg) [reply javascript/javascript1/week1/Homework1/homework.js:3]: Okay, I got it 👍 thank you for the clarification 😊

File: javascript/javascript1/week1/Homework1/homework.js (hunk 2 above)
Comment (Divyajg) [reply javascript/javascript1/week1/Homework1/homework.js:7]: Oops 😬

File: javascript/javascript1/week1/Homework1/homework.js (hunk 3 above)
Comment (Divyajg) [reply javascript/javascript1/week1/Homework1/homework.js:8]: Yeah, didn't noticed. It is "you"

File: javascript/javascript1/week1/Homework1/homework.js (hunk 5 above)
Comment (Divyajg) [reply javascript/javascript1/week1/Homework1/homework.js:26]: Okay 👍

File: javascript/javascript1/week1/Homework1/homework.js (hunk 6 above)
Comment (Divyajg) [reply javascript/javascript1/week1/Homework1/homework.js:53]: Thank you 😊

**Comment by Divyajg** [comment]
Thank you, I am clear with using let and const.

**Pull request merged**